tests/unit/memefficiency.tcl

```diff
@@ -404,6 +404,111 @@ run_solo {defrag} {
             r save ;# saving an rdb iterates over all the data / pointers
         } {OK}
 
+        test "Active defrag pubsub: $type" {
+            r flushdb
+            r config resetstat
+            r config set hz 100
+            r config set activedefrag no
+            r config set active-defrag-threshold-lower 8
```

Comment:
The reason I raised the threshold(comprare to `script eval test`) was due to c->pubsub memory fragmentation, since the used memory is 30m after deleting keys, so it's reasonable to achieve a 7% fragmentation rate.